pkg/crypto/tinkcrypto/unwrap_support_test.go

```diff
@@ -38,9 +38,6 @@ func Test_ExtractPrivKey(t *testing.T) {
 	require.EqualError(t, err, "extractPrivKey: can't extract unsupported private key 'type.googleapis.com/"+
 		"google.crypto.tink.AesGcmKey'")
 
-	_, err = extractPrivKey(&keyset.Handle{})
```

Comment:
Passing an empty keyset handle causes Tink to do a panic now instead of throwing an error

Reply:
ok, this is fine then

Reply:
Updated to check for the panic now using require.PanicsWithValue